app/controllers/v1/inventories_controller.rb

```diff
@@ -1,12 +1,20 @@
 class V1::InventoriesController < ApplicationController
-
   before_action :authenticate_user!
 
   def index
     @inventories = Inventory.joins('LEFT OUTER JOIN inventory_members ON inventory_members.inventory_id = inventories.id')
                        .where('inventories.owner_id = ? OR inventory_members.user_id = ?', current_user.id, current_user.id)
   end
 
+  def show
+    @inventory = Inventory.where(id: params[:id]).first
```

Comment:
We might collide here...I've got some work that does something very similar, although I may wind up accepting this since it behaves better when nothing comes back.

One note to make:  Is there a reason we would prefer `Inventory.where(...)` as opposed to `Inventory.find(...)`?